### PR TITLE
Add CXXOBJS recipe in frameworks

### PR DESCRIPTION
**Pull request by junmin-kim**
frameworks .cxx files can be compiled

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: framework/Makefile [hunk 1]
```diff
@@ -40,9 +41,10 @@ $(foreach BDIR, $(BUILDIRS), $(eval $(call Add_Framework,$(BDIR))))
 
 AOBJS = $(ASRCS:.S=$(OBJEXT))
 COBJS = $(CSRCS:.c=$(OBJEXT))
+CXXOBJS = $(CXXSRCS:.cxx=$(OBJEXT))
```
Comment (sunghan-chang) [inline framework/Makefile]: Why cxx? not cpp? What is common name for c++?

Comment (junmin-kim) [reply framework/Makefile]: We can support both cxx and cpp by using
`CXXOBJS = $(patsubst %.cxx,%$(OBJEXT),$(patsubst %.cpp,%$(OBJEXT),$(CXXSRCS)))`,
but what if CXXSRC have same name and different suffix?
ex) a.cpp a.cxx

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**